docs/howto/deployment/asgi/granian.txt

```diff
@@ -0,0 +1,37 @@
+==============================
+How to use Django with Granian
+==============================
+
+Granian_ is a Rust-based high-performance HTTP server for Python applications,
+supporting ASGI, WSGI, and RSGI interfaces.
```

Comment:
This invites the question "what is RSGI, and does Django support it"? I'd avoid mentioning it.

Reply:
Thank you for the review, Jacob. I agree that it could be ambiguous, so I’ve revised it.